vlib/v/tests/skip_unused/struct_field_none.vv

```diff
@@ -0,0 +1,10 @@
+import json
+
+pub struct Struct {
+	version ?int // = none
+}
+
+fn main() {
+	s := Struct{}
+	println('${json.encode(s)}')
```

Comment:
hm, that is weird ... json.encode() is used here, but in the generated code, the error is inside `json__decode__option_int` which is not used by json.encode, but by json.decode, which is not used by the main program.

Reply:
To me, it seems like the fix is in the wrong place - it will ensure that the `builtin` `_option_none` is always available to cgen, no matter if it is used or not.

Reply:
But we already do the same for `_option_ok` and `_option_clone`.

Reply:
Yes, we do the same for `_option_ok` and `_option_clone` too, and they are not used by most short programs either.

I think, that we should work on improving markused, so that the use of options is tracked, and add those only only when they are used as well.

Reply:
Right. 💯